src/pages/settings/Wallet/WalletPage/WalletPage.tsx

```diff
@@ -64,29 +82,27 @@ function WalletPage({bankAccountList, cardList, fundList, isLoadingPaymentMethod
     });
     const [showConfirmDeleteModal, setShowConfirmDeleteModal] = useState(false);
 
-    const hasBankAccount = !_.isEmpty(bankAccountList) || !_.isEmpty(fundList);
+    const hasBankAccount = !_.isEmpty(bankAccountList) ?? !_.isEmpty(fundList);
     const hasWallet = !_.isEmpty(userWallet);
-    const hasActivatedWallet = _.contains([CONST.WALLET.TIER_NAME.GOLD, CONST.WALLET.TIER_NAME.PLATINUM], userWallet.tierName);
+    const hasActivatedWallet = ([CONST.WALLET.TIER_NAME.GOLD, CONST.WALLET.TIER_NAME.PLATINUM] as string[]).includes(userWallet?.tierName ?? '');
     const hasAssignedCard = !_.isEmpty(cardList);
     const shouldShowEmptyState = !hasBankAccount && !hasWallet && !hasAssignedCard;
 
-    const isPendingOnfidoResult = lodashGet(userWallet, 'isPendingOnfidoResult', false);
-    const hasFailedOnfido = lodashGet(userWallet, 'hasFailedOnfido', false);
+    const isPendingOnfidoResult = userWallet?.isPendingOnfidoResult ?? false;
+    const hasFailedOnfido = userWallet?.hasFailedOnfido ?? false;
 
     const updateShouldShowLoadingSpinner = useCallback(() => {
         // In order to prevent a loop, only update state of the spinner if there is a change
-        const showLoadingSpinner = isLoadingPaymentMethods || false;
+        const showLoadingSpinner = isLoadingPaymentMethods ?? false;
```

Comment:
?

Reply:
I think I wasn't clear with this one.

I feel that it makes little sense with the argument declaration like...

```ts
..., isLoadingPaymentMethods = true, ...
```

Is this default `true` value correct? How did you come up with it?